magento/magento2
# Add validation in catalog rule and shopping cart rule form

### Description (*)


Some fields in Catalog Rule and Shopping Cart Rule form should have more validations (Priority, Uses per Coupon, Uses per Customer, Discount Amount, Maximum Qty Discount is Applied To,  Discount Qty Step (Buy X))

### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. Go to backend
2. Marketing-> Cart Price Rule -> Add New Rule
  Marketing-> Catalog Price Rule -> Add New Rule
3. Check validation in some fields (Priority, Uses per Coupon, Uses per Customer, Discount Amount, Maximum Qty Discount is Applied To,  Discount Qty Step (Buy X))

### Questions or comments


### Contribution checklist (*)
